index.js

```diff
@@ -133,6 +138,55 @@ Shopify.prototype.request = function request(url, method, key, params) {
   });
 };
 
+/**
+ * Updates GraphQL API call limits.
+ *
+ * @param {String} throttleStatus The throttle status returned in the GraphQL response, under "extensions"
+ * @private
+ */
+Shopify.prototype.updateGraphqlLimits = function updateGraphqlLimits(throttleStatus) {
+  const callGraphqlLimits = this.callGraphqlLimits;
+
+  callGraphqlLimits.remaining = throttleStatus.currentlyAvailable;
+  callGraphqlLimits.current = throttleStatus.maximumAvailable - throttleStatus.currentlyAvailable;
```

Comment:
Ditto, I will not add more comments like this but there are more occurrences also in the tests.